recipe/meta.yaml

```diff
@@ -61,7 +78,8 @@ source:
     folder: opengl32sw                                                                                            # [win64]
 
 build:
-  number: 0
+  number: {{ build_number }}
+  string: {{ build_number }}{{ dev_nature }}
```

Comment:
Use the pkg hash here.

Reply:
For my debugging purposes I need to switch this depending on `dev_nature`.

Reply:
i.e `{% if dev_nature != '' %}` .. around it or something like that. `dev_nature` will not be set unless forced via the command line.

Reply:
I meant use the pkg hash in addition. Something like `{{ PKG_HASH }}_{{ build_number }}{{ dev_nature }}`

Reply:
OK, cool